srcpkgs/godot/template

```diff
@@ -46,9 +43,10 @@ do_install() {
 	vinstall ${FILESDIR}/godot.desktop 644 /usr/share/applications/
 	vinstall ${wrksrc}/icon.png 644 /usr/share/pixmaps/ godot.png
 
+
 	case "$XBPS_TARGET_MACHINE" in
-		x86_64*|aarch64*) vbin bin/godot.x11.opt.tools.64 godot;;
-		ppc64*) vbin bin/godot.x11.opt.tools.ppc64 godot;;
-		*) vbin bin/godot.x11.opt.tools.32 godot;;
+		x86_64*|aarch64*) vbin bin/godot.linuxbsd.editor.x86_64 godot;;
```

Comment:
This sounds wrong to me...

Reply:
What do you mean with this sounds wrong?

I don't see an issue here?

Reply:
Why would the arm64 binary be called x86_64?

Reply:
Oh now I see what you mean...

Yeah well that is indeed a problem.

I'll change it. Thank you for bringing that to my attention. :)

Reply:
Fixed in the new commit.